code/game/gamemodes/clock_cult/clock_structures/ark_of_the_clockwork_justicar.dm

```diff
@@ -242,6 +242,19 @@
 				var/obj/structure/destructible/clockwork/massive/ratvar/R = new(startpoint)
 				var/turf/T =  locate(round(world.maxx * 0.5, 1), round(world.maxy * 0.5, 1), ZLEVEL_STATION_PRIMARY) //approximate center of the station
 				R.forceMove(T)
+				SSticker.force_ending = TRUE
+				var/x0 = T.x
+				var/y0 = T.y
+				for(var/I in spiral_range_turfs(255, startpoint, tick_checked = TRUE))
```

Comment:
should be T rather than startpoint.

Reply:
why create him at startpoint then forceMove him to T though?